rfd/xxxx-gcp-server-discovery.md

```diff
@@ -254,6 +247,24 @@ executed over SSH, with access managed with keys in the VM's
 metadata. There is no permission specifically for running commands; the most
 limiting one is `compute.instances.setMetadata`.
 
+## IaC
+
+### Terraform
+
+The `gcp` join method extents `ProvisionTokenV2`, so no extra work will be
```

Comment:
Is this true ? Will the new allow fields not need configuring ?

Reply:
The new fields for Azure join didn't need configuring. I doubt GCP would be any different.

Reply:
👍  Ok cool - wasn't sure if this was auto-generated or not - it appears to be.